### DISCUSSION TITLE
Eslint presets for typegoose?

### DISCUSSION THREAD

**Discussion by vasyl97kharchenko** [typegoose/typegoose#833] [Q&A]
Is there any eslint presets for typegoose? Or is there any eslint setup for typegoose project for better error coverage?

**Comment by hasezoey** [comment]
> Is there any eslint presets for typegoose?

there is no specific eslint preset specially for typegoose, typegoose is basically just classes, a normal preset should be just enough

> Or is there any eslint setup for typegoose project for better error coverage?

aside from stylistic things (like indent) there are the typescript warnings / errors, aside from those, there are only some runtime errors which cannot be expressed with typescript

---

you can look at [typegoose's eslintrc](https://github.com/typegoose/typegoose/blob/e8470f4b4621c579d26c70d0fb76a68fc3909042/.eslintrc.js) for how typegoose does it, but there is not much in it
